tests/test_cli.py

```diff
@@ -15,25 +15,23 @@
 import rsa
 import rsa.cli
 import rsa.util
+from rsa._compat import PY3
 
-if sys.version_info[0] < 3:
-    def make_buffer():
-        return BytesIO()
 
-
-    def get_bytes_out(out):
-        # Python 2.x writes 'str' to stdout:
-        return out.getvalue()
-else:
-    def make_buffer():
+def make_buffer():
```

Comment:
I try to prioritize readability and `DRY`ness instead of (over)optimization. Even if these functions don't share logic, I prefer to have a unique point of "failure", one docstring to maintain and shared common blocks inside the function if that's possible.

In my opinion, those advantages will make devs to save a lot more time when trying to contribute to the project or follow a stack trace than one more comparison per call will ever provide. Does that make any sense?